packages/admin-stories/src/docs/components/ColorPicker/ColorPicker.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Canvas, Story } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/Color Picker" />
+
+# Color Picker
+
+<Canvas>
+    <Story id="stories-components-color-picker-color-picker--color-picker" />
```

Comment:
The id-system is a bit of a pain to handle...
`storiesOf("stories/components/Color Picker/Color Picker", module).add("Color Picker", () => {
` gets `stories-components-color-picker-color-picker--color-picker`

But there is now way around that I guess.

Reply:
Yes thats a kind of pain. I found no way around that. If we put stories inline of mdx there is no typescript support and the mdx files get bloen up and there is no other way to embeed a story in a mdx file